### PR TITLE
feature/Home-Complete

### PR DESCRIPTION
**Pull request by usaidam1n**
#### Implementation:
- Implemented react native navigation.
- Added and linked Roboto font.
- Added folder structure.
- Added necessary styling.

#### Testing: 
- Android testing on OnePlus 8 and Google nexus.
- iOs testing on iPhone 8 and iPhone X.
- Tested before committing.

___

#### Screenshot/Video:

**Android:**

Screen Shot 2021-09-16 at 00 16 20

https://user-images.githubusercontent.com/46266811/133507097-83d160b4-5c96-436d-9627-241b3bb0ad60.mov

https://user-images.githubusercontent.com/46266811/133507211-86e30247-7a15-44a0-932f-4bfdf2463a00.mp4
___

**IOS:**

Screen Shot 2021-09-16 at 00 03 29
Screen Shot 2021-09-15 at 23 59 42

https://user-images.githubusercontent.com/46266811/133507584-76a32dc7-dbde-40cf-8961-1680fc87b04e.mov

https://user-images.githubusercontent.com/46266811/133507495-56b0a3d7-d461-43d3-b768-eccccdd3798b.mov

### CODE REVIEW

File: android/app/src/main/java/com/poweroutrage/MainActivity.java [hunk 1]
```diff
@@ -15,8 +15,8 @@ protected String getMainComponentName() {
     return "PowerOutrage";
   }
   @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        SplashScreen.show(this, true);
-        super.onCreate(savedInstanceState);
-    }
+  protected void onCreate(Bundle savedInstanceState) {
+    SplashScreen.show(this, true);
+      super.onCreate(null);
```
Comment (saamerm) [inline android/app/src/main/java/com/poweroutrage/MainActivity.java:20]: @usaidpeerzada is there a reason you are not passing in the `savedInstanceState`?

**Review by saamerm — CHANGES_REQUESTED** [review]
@usaidpeerzada  Apart from the comment, do we need the Roboto-Regular.otf & Roboto-Regular.otf files? Are we using them anywhere?

File: src/screens/HomeScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import {View, Image, Text, Dimensions, StyleSheet, SafeAreaView, Button, TouchableOpacity} from 'react-native';
+import {useNavigation} from '@react-navigation/native';
+
+const windowWidth = Dimensions.get('window').width;
+const windowHeight = Dimensions.get('window').height;
+export default function HomeScreen() {
+    const navigation = useNavigation();
+    return (
+        <SafeAreaView>
+            <Image source={require('../assets/images/splash_logo.png')} style={styles.ImageStyle} resizeMode='contain'/>
```
Comment (saamerm) [inline src/screens/HomeScreen.tsx:11]: Can we just use import instead of a require? @usaidpeerzada

**Review by saamerm — CHANGES_REQUESTED** [review]

File: src/screens/HomeScreen.tsx (hunk 2 above)
Comment (usaidam1n) [reply src/screens/HomeScreen.tsx:11]: In the rn docs I found two ways to get an image, if it is locally downloaded then require , if we get it from some url then uri:

I will check if there is a way to import.

File: android/app/src/main/java/com/poweroutrage/MainActivity.java (hunk 1 above)
Comment (usaidam1n) [reply android/app/src/main/java/com/poweroutrage/MainActivity.java:20]: https://reactnavigation.org/docs/getting-started

**Review by saamerm — APPROVED** [review]

**Pull request merged**